src/Main.java

```diff
@@ -1,21 +1,82 @@
 import java.util.HashSet;
 import java.util.Objects;
+import java.util.Random;
 import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
 
-        // Hier mag je je code scrijven voor de hoofd-opdracht
+        // Hier mag je je code schrijven voor de hoofd-opdracht
+        Integer[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+        String[] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul"};
 
-        /* deze regel mag je weg halen voor de bonus opdracht. Onderstaande code is voor de bonus opdracht.
-        HashSet<Integer> secretnumber = randomnumbergenerator();
+        Translator translator = new Translator(alphabetic, numeric);
+
+        boolean play = true;
+        String ongeldig = "ongeldige invoer";
+        Scanner scanner = new Scanner(System.in);
+
+        while (play) {
+            // Je kan hierboven play == true verkorten naar 'play' omdat deze altijd checkt of iets 'true' is. Dus je hoeft het niet specifiek te benoemen.
```

Comment:
Ik ben een beetje in de war hierzo, hebben andere mensen je ook feedback gegeven? Of heb je jezelf feedback gegeven?